recipes/gperftools/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class GperftoolsConan(ConanFile):
+    name = "gperftools"
+    license = "BSD-3-Clause License"
+    homepage = "https://github.com/gperftools/gperftools"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "gperftools: originally Google Performance Tools"
+    topics = ("malloc", "profiler")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"fPIC": False, "shared": True}
+    autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
```

Comment:
gperftools [provides a cmake script](https://github.com/gperftools/gperftools/blob/master/CMakeLists.txt),
why not use that?

Reply:
I was following the build instructions from the gperftools [INSTALL](https://github.com/gperftools/gperftools/blob/master/INSTALL) page. Also the cmake file has the following warning 
```cmake support is very preliminary. Autotools-based build is the only fully supported build for now.```

Reply:
Okay. autotools will probably be the most tested and guaranteed to have the most configuration options.